tsconfig.json

```diff
@@ -12,6 +12,8 @@
     "noEmit": true
   },
   "include": [
-    "src"
+    "src",
+    "api",
+    "@types"
```

Comment:
What does including `@types` do? Does it work if we remove it?

Also nit: alphabetize